server/src/storage/datastore.ts

```diff
@@ -38,60 +77,109 @@ const extractAndAppendId = (res: Entity) => {
 
 /**
  * A Datastore wrapper that gets a particular entity with the specified Kind and id.
- * @param kind The Kind that is being queried
- * @param id The id of the Entity being queried
+ * If actor is not specified, gets using datastore.
```

Comment:
update 'actor' to 'transaction' in comments

Reply:
@ifhuang Oh I missed that our when editing! I think I will just remove this line since it is kind of repetition for
```
 * @param transaction Transaction that will carry out the retrieval,
 * if not specified, datastore will carry out the operation
```